.github/workflows/pages.yml

```diff
@@ -24,6 +24,12 @@ jobs:
         run: |
           echo $CONDA/bin >> $GITHUB_PATH
       - name: Installing the library
+        env:
+          TIDY3D_USER: ${{ secrets.TIDY3D_EMAIL }}
+          TIDY3D_PASS: ${{ secrets.TIDY3D_PASSWORD }}
+          DISPLAY: 1.0
+          DOCS: True
+          plotter: klayout
```

Comment:
a more descriptive environment variable name would probably be better... i.e. GDSFACTORY_LAYOUT_PLOTTER

Reply:
Thank you Troy!

added  a better variable name